Dashboard/pages/add_product.py

```diff
@@ -0,0 +1,67 @@
+from os import _Environ, environ as CONFIG
+import logging
+
+import streamlit as st
+from psycopg2.extensions import connection
+
+from navigation import make_sidebar
+
+from helpers import (
+    get_connection, can_parse_as_float,
+    get_supported_websites,
+    update_db_to_subscribe
+)
+from load import load_product_data
```

Comment:
Ignore pylint here

Reply:
Do you need the brackets?

Reply:
Yes, otherwise pylint will complain of the line being too long. This is also clearer